docs/platform/reference/eol-for-major-versions.md

```diff
@@ -186,7 +186,7 @@ To support this transition, Aiven has extended support for Apache Kafka 3.8 by o
 
 | Version | Aiven EOL       |
 | ------- | --------------- |
-| 24.8    | To be announced |
+| 25.3    | To be announced |
```

Comment:
The pr looks good, no major complains.

But I just want to clarify
1. should we release this commit after the 25.3 is released?
2. We could actually fill the table with some data. Like 24.8 new service creation is allowed till ~2025-12-11, and since the same data 25.3 creation is supported. Also, we did not plan it, but we should set up 24.8 EOL to something like 2026-03-01